devel/gflags/Portfile

```diff
@@ -5,26 +5,47 @@ PortGroup           github 1.0
 PortGroup           cmake 1.1
 
 github.setup        gflags gflags 2.2.2 v
-revision            0
+revision            1
 
 categories          devel
 maintainers         nomaintainer
 
 description         commandline flags module for C++
 long_description    The gflags package contains a library that implements commandline flags processing. \
-                    As such it's a replacement for getopt(). It has increased flexibility, \
+                    As such it’s a replacement for getopt(). It has increased flexibility, \
                     including built-in support for C++ types like string, and \
                     the ability to define flags in the source file in which \
-                    they're used.
+                    they’re used.
```

Comment:
Sergey, don't change single quotes to these characters. The standard ones are clean single-byte ASCII characters, but the curly variants are multi-byte.

You can see the result by viewing the file via `less`:

```
long_description    The gflags package contains a library that implements commandline flags processing. \
                    As such it<E2><80><99>s a replacement for getopt(). It has increased flexibility, \
                    including built-in support for C++ types like string, and \
                    the ability to define flags in the source file in which \
                    they<E2><80><99>re used.
```

Please revert.

Reply:
In short, avoid using curly apostrophes anywhere in portfiles.

Reply:
Not a big deal though, as it doesn't cause any problems. So no worries if this change has been made elsewhere!

Reply:
> Not a big deal though, as it doesn't cause any problems. So no worries if this change has been made elsewhere!

@mascguy I changed it for a reason, in BBEDit (and I suspect other code editors) it breaks code coloring, since the editor understand that as a quote character of the code. My change not just makes typography correct (a matter of æsthetics), but fixes code coloring in code editor.

Reply:
@mascguy Dropped the commit for BBEdit code coloring fix.